doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -219,7 +219,7 @@ They also support the :attr:`universes(polymorphic)` attributes.
    .. rocqdoc::
 
      Add Parametric Relation (A : Type) : A (@eq A)
-       [reflexivity proved by @refl_equal A]
+       reflexivity proved by (@refl_equal A)
```

Comment:
Better as a rocqtop example.  The syntax error would have been found automatically.